internal/utils.go

```diff
@@ -183,6 +191,26 @@ func getPolicyTemplates(policyConf *types.PolicyConfig) ([]map[string]map[string
 	return policyTemplates, nil
 }
 
+// checkPolicyTypeManifest determine if the manifest is policy-type manifest.
+func checkPolicyTypeManifest(manifest map[string]interface{}) bool {
```

Comment:
This is a little easier to understand in my opinion.
```suggestion
func isPolicyTypeManifest(manifest map[string]interface{}) bool {
```